add tizen-8-0-m1.md release note for public release (#1824)
Samsung/tizen-docs: docs/platform/release-notes/tizen-8-0-m1.md

### System (kernel and system framework)

#### New and changed features

- System and resource management
  - Multi-user user-add function has been improved by fixing smack/permission exception case.
  - Multi-user user-get-list latency issue has been fixed by getting the list from directory.
  - Ready-state callback of multi-user service has been supported to reduce booting-time.
  - Process OOM_FIXED_APPS feature has been restored.
  - Process CPU affinity feature has been restored.
- Device and sensor management
  - Input event on/off function for input device has been supported.
  - Display white balance function has been supported.
  - Display rotation function has been supported.
  - Power suspending and resuming callback function have been supported.
  - Power suspending cancellation function has been supported.
  - Power lock status callback has been supported.
  - Power wakeup reason function has been supported.
  - Uart flush and drain function have been supported.
  - Sensor listener attribute getter function has been supported.
  - Sensor attribute setter/getter functions have been supported.
  - Sensitivity attribute of Proximity sensor has been supported.

>   - Sensord signal handling has been improved by removing race conditions.

Author: Hello~, actually sensord is specific module name(proper name) which means "sensor-daemon"
you can search easily by googling also.
what should I do? please let me know :-)

  - tzdata
    - Version 2021e has been upgraded to version 2022g.
  - update-alternatives
    - Version 1.20.5 has been upgraded to version 1.21.18.
  - uthash
    - Version 1.9.7 has been upgraded to version 2.3.0.
  - vim
    - Version 8.2.2173 has been upgraded to version 8.2.5172.
  - xmlto
    - Version 0.0.25 has been upgraded to version 0.0.28.
  - xz
    - Version 5.2.5 has been upgraded to version 5.4.0.
  - yasm
    - Version 1.2.0 has been upgraded to version 1.3.0.
  - zlib
    - Version 1.2.11 has been upgraded to version 1.2.13.
  - zstd
    - Version 1.4.5 has been upgraded to version 1.5.2.

### Application framework

#### New and changed features

- Application Manager
  -  C# API that terminates without restarting application has been added.
- Package Manager
  - New feature has been ready to remove user data from other applications.
- App-control 
  - Launch-bounds APIs have been provided.
  - Extended appid for multi-instance has been introduced.
- TIDL
  - The TIDL compiler has been updated to generate code that use Cion API for Dart language
- Tizen Database Connectivity (TDBC)
  - New feature has been ready to access  a database.

>     - Provides an C# API which defines how a client may access a database

Reviewer: ```suggestion
    - Provides an C# API which defines how a client may access a database.

```

  - The name of e_desk_group has been changed to e_desk_area.
  - The server protocol implementation of wtz_screen and wtz_splitscreen interfaces has been included.
  - The server protocol implementation of wtz_shell interface has been included.
  - The policies for Multi-Windows management have been added.
    - Those are Smart Launch, Snap Window, All Minimize and Smart Rotation.
  - The wheel event in touchpad has been supported.
- TBM
  - Parallelization function to improve the authentication speed from TBM has been added.
- TPL-EGL
  - The API to set the frontbuffer rendering for each surface has been provided.
  - The API to check if a surface has a fence sync has been provided.
- Mesa
  - The version of Mesa has been upgraded to 22.3.5.
- Vulkan
  - The version of SPIRV-Cross, SPIRV-Headers and SPIRV-Tools has been upgraded to 1.3.239.
  - The version of Vulkan-Headers, Vulkan-Hpp, Vukan-Loader and Vulkan-ValidationLayers has been upgraded to 1.3.240.
  - The version of glslang has been upgraded to 1.3.239.
- Libinput
  - The version of Libinput has been upgraded to 1.22.0.
- Libxkbcommon
  - The version of Libxkbcommon has been upgraded to 1.5.1.
- TTS Framework

>   - Supports root daemon as TTS client

Reviewer: ```suggestion
  - Supports root daemon as TTS client.

```

- Bluetooth
  - New APIs for GATT client connecting and disconnecting have been added.
  - New APIs for GATT server connection state changed callback have been added.
  - New API for setting advertising flags has been added.
  - New APIs for setting advertising custom name have been added.
- Open source
  - Curl version has been upgraded to 7.86.0.
  - Wpa_supplicant version has been upgraded to 2.10.

#### Fixes

- Network
  - The logic to store profile extension information has been fixed.
  - The logic for Netlink scan has been fixed.
  - The logic to calculate scores for AP connections has been fixed.
  - The logic for background scan has been fixed.
  - The logic to monitor the status of the Wi-Fi module has been fixed.
- Bluetooth
  - The RSSI parsing issue from discovery has been fixed.
  - Discovery busy state handling logic has been fixed.
  - Adapter state check logic has been fixed.
  - EIR manufacturer data handling logic has been fixed.
  - Bonded devices response not coming issue has been fixed.
  - Infinite callback issue in EOF case has been fixed.

### Security

#### New and changed features

- Privacy privilege manager

>   - Privacy privilege manager feature (PPM) has been deprecated.

Reviewer: ```suggestion
  - Privacy privilege manager (PPM) feature has been deprecated.

```

### Service framework

#### Fixes

- Account Framework
  - Potential defects have been fixed.
- Sync-Manager
  - Potential defects have been fixed.

### Web framework

#### New and changed features

- Web Engine
  - Open source chromium 108 version base has been applied.
  - OzonePlatform for EFL has been added.
  - Ozone EFL based Onscreen/Offscreen rendering mode has been supported.
  - Offscreen rendering mode based NUI feature has been supported.

>   - ESPP based Media Renderer has been added.

Reviewer: ```suggestion
  - ESPP based media renderer has been added.

```

- Web Runtime
  - Open source electron 22.0.0 version has been applied.

#### Fixes

- Stability issue in multi windows scenario has been fixed.
- Media permission issue on WRT has been fixed.

### Lightweight web solution

#### New and changed features

- Lightweight Web Engine
- Web Engine
- Service Worker lifecycle has been supported.
- Caching responses on network requests in the Service Worker has been supported.
- Maplike declaration has been added.
- JavaScript Engine

> - Ecmascript feature (from es2022) has been added.

Reviewer: ```suggestion
- ECMAscript feature (from ES2022) has been added.

```

- Network loader defects have been fixed.
- Memory leak defects have been fixed.
- Url parsing defects have been fixed.
- History management defects have been fixed.

### Tizen .NET

#### New and changed features

- TizenFx
  - Tizen.Content.MediaContent
    - Deprecated properties have been removed from MediaInfoColumns class.
    - Deprecated classes (StorageType/Storage/StorageCommand) have been removed.
  - Tizen.Applications.PackageManager
    - ClearUserData method has been added.
  - Tizen.Applications.RPCPort
    - New constructor has been added to Parcel class.
  - Tizen.Nlp has been deprecated.
  - Tizen.Security.PrivacyPrivilegeManager has been deprecated.
  - Tizen.Security.DevicePolicyManager has been deprecated.
  - Tizen.Multimedia.Camera
    - New setting APIs have been added.
  - Tizen.Data.Tdbc has been added.
  - Tizen.Applications.Common
    - New resource control APIs have been added.
  - Tizen.Multimedia.Remoting
    - WebRTCStatisticsProperty enum have been updated.
  - Tizen.Multimedia.Vision
    - New QR APIs have been added.
  - Tizen.Content.Download
    - New caching APIs have been added.

### Toolchain

#### New and changed features

- GCC

>   - arm target changes has been back-ported from releases/gcc-9.

Reviewer: ```suggestion
  - ARM target changes have been back-ported from releases/gcc-9.

```

  - Libsanitizer: AddressSanitizer overhead reduction option has been added.
- Glibc

>   - Timezone: updated from tzcode 2020a 

Reviewer: ```suggestion
  - Timezone: updated from tzcode 2020a.

```

- Binutils
  - Packaging: keep shared libraries for libbfd & libopcodes.

#### Fixes

- GCC
  - Bugfix PR c++/61414 patchset has been applied.
  - Bugfix PR middle-end/98189, 95886 patchset has been applied.
- Glibc

>   - Static TLS(Thread Local Storage) block memory allocation bugfix patchset has been applied.

Reviewer: ```suggestion
  - Static TLS (Thread Local Storage) block memory allocation bugfix patchset has been applied.

```

  - CVE bugs are fixed
    - CVE-2022-23219
    - CVE-2022-23218
    - CVE-2021-38604
    - CVE-2021-35942
    - CVE-2021-3326
    - CVE-2021-27645
    - CVE-2020-27618
    - CVE-2021-3999
  - AddressSanitizer build error for aarch64 architecture has been fixed.

### Machine learning

#### New and changed features

- Machine Learning (ML) API updates
  - ML.Service API Updates
    - Add new APIs to separately manage ML model files. These APIs allow ML applications to exploit the latest ML model deployed without code-level modifications that require re-packaging and re-distribution of the whole application.
    - AI Inference offloading between Tizen and TizenRT has been improved. The binary size of TizenRT is optimized (< 100kB). 
  - ML.Inference API Updates
    - Add new API to use extended rank limit. To maintain the backward compatibility, the default rank limit is 4. The extended rank is only used when new API is called.

>     - Support TensorFlow-Lite v2.11.0 by default.

Reviewer: ```suggestion
    - Support TensorFlow Lite v2.11.0 by default.

```

  - ML.Training API Updates
    - Add Identify Layer as a utility layer which flows everthing as it is.

>     - Add Learning Rate Scheduling Features.
> - Step Learning Rate Scheduling
> - Exponent Learning Rate Scheduling

Reviewer: ```suggestion
    - Add learning rate scheduling features.
- Step learning rate scheduling
- Exponent learning rate scheduling

```

    - Add API to get the weight data
- NNStreamer updates
  - NNStreamer has been upgraded to version 2.3.0.
  - Support for large-size model of the NNStreamer pipeline has been added.
    - The number of rank limit has been increased up to 8.
    - The number of tensor limit has been increased up to 100.
  - Support for the model storage of Device MLOps has been added.
    - Model management feature such as register, fetch active model, update, and delete has been added. Using this feature, ML applications can share their model with other applications.
  - Pipeline based data repository and training feature has been added.
    - New NNStreamer elements such as datareposrc, datareposink, and tensor_trainer have been added.
    - Support for the training in AI pipeline has been added.
- NNTrainer updates

>   - Support proactive swap which utilize secondary storage for less memory consumption
>     - Add Cache Pool / Cache Loader / Cache Element classes 
>     - Update Memory pool and planner for better utilization
>   - Added Execution order and memory usage tarcing for debugging
>   - Added TFLite exporter which generate inference model for tflite (unstable)

Reviewer: ```suggestion
  - Support proactive swap which utilizes secondary storage for less memory consumption.
    - Add Cache Pool/Cache Loader/Cache Element classes.
    - Update Memory pool and planner for better utilization.
  - Added Execution order and memory usage tarcing for debugging.
  - Added TFLite exporter which generates inference model for tflite (unstable).

```
